src/components/desktop/Header.js

```diff
@@ -201,9 +213,23 @@ const Header = () => {
             </StyledDropdown>
           </li>
           <li>
-            <button>
-              <User /> {t('Login')}
-            </button>
+            {authToken ? (
+              <StyledDropdown
+                label={
+                  <StyledUser>
+                    <User height={'15px'} />{' '}
+                    {authToken ? 'Arman Rafati' : t('Login')}
```

Comment:
Lets add a TODO here also, and @code-arman can use somebody else's name here (John Doe, etc) if you care